yb-voyager/cmd/assessMigrationCommandBulk.go

```diff
@@ -292,6 +278,21 @@ func generateBulkAssessmentReport(dbConfigs []AssessMigrationDBConfig) error {
 	return nil
 }
 
+func checkMigrationAssessmentForConfig(dbConfig AssessMigrationDBConfig) bool {
+	if !utils.FileOrFolderExists(dbConfig.GetAssessmentReportPath()) {
+		return false
+	}
+
+	// Note: Checking for report existence might be sufficient
+	// but checking MSR as an additionally covers for edge cases if any.
+	initMetaDB(dbConfig.GetAssessmentExportDirPath())
```

Comment:
How about using this instead? Let's try and avoid using global variables wherever possible. 
Basically, don't init the global metadb variable with the passed exportdir, create a new metadb explicitly for each case.
```
CreateAndInitMetaDBIfRequired
 metadb.NewMetaDB(exportDir)
```

Reply:
I didn't get this completely.

For the initMetaDB() function, i have made the `exportDir` info local, by passing it as a func arg.
`initMetaDB()` --> `initMetaDB(exportDir)`

Reply:
yes true, but if you look at the `initMetaDB` function:
```
metaDB, err = metadb.NewMetaDB(exportDir)
```
here, `metaDB` is a global variable. So, every time, you are setting the global metaDB variable, which can be error-prone. You could probably modify `initMetaDB` to return a metaDB instance instead of saving it in the global variable. (in addition to making the exportDid arg local)

Reply:
got it